src/printer.js

```diff
@@ -674,23 +673,23 @@ function genericPrintNoParens(path, options, print) {
     case "StringLiteral":
     case "Literal":
       if (typeof n.value === "number") return n.raw;
-      if (typeof n.value !== "string") return fromString(n.value, options);
+      if (typeof n.value !== "string") return "" + n.value;
```

Comment:
I manually went through every call site and added `"" +` when it wasn't clear if it was a string.